docs/SPIR-V.rst

```diff
@@ -3753,6 +3753,10 @@ implicit ``vk`` namepsace.
   
     uint64_t ReadClock(in uint scope);
     uint     RawBufferLoad(in uint64_t deviceAddress);
+    void     RawBufferLoadToParam(out T output, in uint64_t deviceAddress);
```

Comment:
One idea for alternative name: `RawBufferLoadInto`.

Reply:
Done